### PR TITLE
Bryjbrown islandora 2185

### PR DESCRIPTION
**Pull request by rosiel**
@bryjbrown 

Fixes the subTitle problem, and stops leaving "temporary" files around.

### CODE REVIEW

File: islandora_scholar.drush.inc [hunk 1]
```diff
@@ -141,37 +152,60 @@ EOQ;
...
   $action = drush_get_option('action', variable_get('islandora_doi_handle_imported_metadata', FALSE));
-  if (!in_array($action, array('strip', 'encode'))) {
-    drush_print("This function requires an option, since the GUI variable of how
+  $audit = drush_get_option('audit', FALSE);
+  $dry_run = drush_get_option('dry-run', FALSE);
+  $modify_my_data = drush_get_option('modify-my-data', FALSE);
+
+  // Require one and only one of --audit, --dry-run, --encode.
+  if (!($audit || $dry_run || $modify_my_data)) {
+    drush_print("This function requires one option of the following: --audit, --dry-run, --encode.");
```
Comment (d-r-p) [inline islandora_scholar.drush.inc]: `s/encode/modify-my-data/`

File: islandora_scholar.drush.inc [hunk 2]
```diff
@@ -141,37 +152,60 @@ EOQ;
...
+    drush_print("USAGE:");
+    drush_invoke('help',['islandora-scholar-fix-doi-html-in-mods']);
+    exit;
+  }
+  if (($audit && $dry_run) || (($audit || $dry_run) && $modify_my_data)) {
+    drush_print("This function requires ONLY one option of the following: --audit, --dry-run, --encode.");
```
Comment (d-r-p) [inline islandora_scholar.drush.inc]: `s/encode/modify-my-data/`

File: islandora_scholar.drush.inc [hunk 3]
```diff
@@ -141,37 +152,60 @@ EOQ;
...
+    drush_print("USAGE:");
+    drush_invoke('help',['islandora-scholar-fix-doi-html-in-mods']);
+    exit;
+  }
+
+  // Require an action if --dry-run or --encode is selected.
```
Comment (d-r-p) [inline islandora_scholar.drush.inc]: `s/encode/modify-my-data/`

File: islandora_scholar.drush.inc [hunk 4]
```diff
@@ -141,37 +152,60 @@ EOQ;
...
+  if (($audit OR $dry_run) AND (!in_array($action, array('strip', 'encode')))) {
```
Comment (d-r-p) [inline islandora_scholar.drush.inc]: `s/audit/modify_my_data/`?

Comment (rosiel) [reply islandora_scholar.drush.inc]: yup.

**Pull request merged**